webapp/src/main/webapp/vue-apps/space-form/initComponents.js

```diff
@@ -41,6 +43,7 @@ const components = {
   'space-form-invite-email-list-item': SpaceFormInviteEmailListItem,
   'space-form-invite-user-list-item': SpaceFormInviteUserListItem,
   'space-form-invite-users-input': SpaceFormInviteUsersInput,
+  'space-preview': SpacePreview,
```

Comment:
Please add a specific naming to SpaceFormDrawer for Vue components

```suggestion
  'space-form-preview': SpacePreview,
```